log.js

```diff
@@ -36,8 +36,7 @@ module.exports = function (dir, config, private) {
   let originalGet = log.get
   log.get = function (seq, cb) {
     originalGet(seq, (err, res) => {
-      if (err && err.code === 'flumelog:deleted') cb()
```

Comment:
This was removed because we already handle this in jitdb and the only other place this is used is in EBT index and actually if that happens then this means that you delete a message from an author but not the whole feed and this scenario shouldn't be supported in EBT anyway.